scripts/istat/geos/preprocess.py

```diff
@@ -0,0 +1,96 @@
+# Lint as python3
```

Comment:
Do you mind adding a header for all the code files? Can copy this one: https://github.com/datacommonsorg/data/blob/master/scripts/us_fed/treasury_constant_maturity_rates/generate_csv_and_mcf.py#L1-L13

Reply:
done